examples/standard_platformer/lib/main.dart

```diff
@@ -93,7 +93,7 @@ class ExamplePlatformerLeapGame extends LeapGame
     );
 
     player = Player();
-    world.add(player = Player());
```

Comment:
Nit: I would keep this line, and instead, remove the one above

Reply:
Hm, but why? 😀

Personally not a fan of the assignment side effects

Reply:
Just cause one line less and we avoid the ! 

But this is just a minor bit, totally not blocking for the pr :)